Manito/Manito/Screens/Letter/UIComponent/CreateLetterPhotoView.swift

```diff
@@ -0,0 +1,225 @@
+//
+//  CreateLetterPhotoView.swift
+//  Manito
+//
+//  Created by SHIN YOON AH on 2022/06/13.
+//
+
+import PhotosUI
+import UIKit
+
+import SnapKit
+
+final class CreateLetterPhotoView: UIView {
+
+    typealias alertAction = ((UIAlertAction) -> ())
+    
+    var setSendButtonEnabled: ((_ hasImage: Bool) -> ())?
+    
+    private enum SourceType {
+        case camera
+        case library
+    }
+    
+    // MARK: - ui component
+    
+    private let importPhotosButton: UIButton = {
+        let button = UIButton()
+        button.makeBorderLayer(color: .white)
+        button.clipsToBounds = true
+        button.setImage(ImageLiterals.btnCamera, for: .normal)
+        button.imageView?.contentMode = .scaleAspectFill
+        button.setPreferredSymbolConfiguration(.init(pointSize: 25), forImageIn: .normal)
+        button.tintColor = .white
+        button.backgroundColor = .darkGrey004
+        return button
+    }()
+    private let titleLabel: UILabel = {
+        let label = UILabel()
+        label.text = TextLiteral.letterPhotoViewTitleLabel
+        label.font = .font(.regular, ofSize: 16)
+        return label
+    }()
+
+    // MARK: - property
+
+    private var hasImage: Bool {
+        return self.importPhotosButton.imageView?.image != ImageLiterals.btnCamera
+    }
+    var image: UIImage? {
+        return self.importPhotosButton.imageView?.image
+    }
+
+    // MARK: - init
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        self.setupLayout()
+        self.setupButtonAction()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    // MARK: - func
+    
+    private func setupLayout() {
+        self.addSubview(self.titleLabel)
+        self.titleLabel.snp.makeConstraints {
+            $0.top.leading.equalToSuperview()
+        }
+        
+        self.addSubview(self.importPhotosButton)
+        self.importPhotosButton.snp.makeConstraints {
+            $0.top.equalTo(self.titleLabel.snp.bottom).offset(17)
+            $0.leading.trailing.bottom.equalToSuperview()
+            $0.height.equalTo(209)
+        }
+    }
+    
+    private func setupButtonAction() {
+        let photoAction = UIAction { [weak self] _ in
+            self?.presentActionSheet(actionTitles: self?.actionTitles() ?? [],
+                                     actionStyle: self?.actionStyle() ?? [],
+                                     actions: self?.alertActions() ?? [])
+        }
+        self.importPhotosButton.addAction(photoAction, for: .touchUpInside)
+    }
+
+    private func actionTitles() -> [String] {
+        return self.hasImage ? [TextLiteral.letterPhotoViewTakePhoto, TextLiteral.letterPhotoViewChoosePhoto, TextLiteral.letterPhotoViewDeletePhoto, TextLiteral.cancel] : [TextLiteral.letterPhotoViewTakePhoto, TextLiteral.letterPhotoViewChoosePhoto, TextLiteral.cancel]
+    }
```

Comment:
p5;

딱히 정해진건 아니지만 가독성을 위해
```suggestion
    private func actionTitles() -> [String] {
        return self.hasImage
        ? [TextLiteral.letterPhotoViewTakePhoto,
           TextLiteral.letterPhotoViewChoosePhoto,
           TextLiteral.letterPhotoViewDeletePhoto,
           TextLiteral.cancel]
        : [TextLiteral.letterPhotoViewTakePhoto,
           TextLiteral.letterPhotoViewChoosePhoto,
           TextLiteral.cancel]
    }
```
요로코롬 수정하는건 어떨까요 ??

Reply:
으음 네~  `self.hasImage`에서 한 칸 뛰는건 싫어서 고건 붙여서 쓰겠습니다. ٩(　ᐕ)و 
[7a368f3](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/379/commits/7a368f320377838e1c4a79c00d3dd108ecbe5a93)

Reply:
ㅋㅋㅋㅋㅋ 좋습니다 !